models/migrations/20190829140646_user_library.js

```diff
@@ -0,0 +1,23 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('library', tbl => {
+    tbl.increments();
+    tbl
+      .foreign('users_to_library')
+      .reference('user_id')
+      .inTable('users')
+      .onDelete('CASCADE')
+      .onUpdate('CASCADE')
+      .notNullable();
+    tbl
+      .foreign('library_to_books')
+      .reference('books_id')
+      .inTable('books')
+      .onDelete('CASCADE')
+      .onUpdate('CASCADE')
+      .notNullable();
+  });
```

Comment:
Same thing as above so if there is an error, the solution is above.  If it works as intended, nevermind then!